vm-setup/roles/v1aX_integration_test/tasks/download_image.yml

```diff
@@ -18,7 +18,7 @@
 
           - name: Verify specific centos image containing newer version of cloud-init is downloaded
             get_url:
-              url: "{{ IMAGE_LOCATION_CENTOS }}"
+              url: "{{ IMAGE_LOCATION_CENTOS }}/{{ IMAGE_NAME_CENTOS }}"
               dest: "{{ IRONIC_IMAGE_DIR }}/{{ IMAGE_NAME_CENTOS }}"
```

Comment:
If you change that @smoshiur1237 , would you also change all the occurrences of  `IMAGE_NAME_CENTOS` like this

Reply:
Yes the download file needs optimization. It can have the same name for both OS.